### DISCUSSION TITLE
Process script execution problem

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#903] [Q&A]
Ngrinder configures to execute the process script once in the execution scenario, but the script code ends before it is executed. What is the reason? It can be executed when the code module is debugged and executed
For process script, I have multiple requests in the script, and individual requests need to loop for 100 times, but when setting scenario execution, it stops before 100 times are completed

![image](https://user-images.githubusercontent.com/50607317/192415851-30ed974a-7002-4c23-99af-38a9c69daff6.png)
The script logic looks like this，Set multiple recoders. When the recoder reaches the first method, it will end
![image](https://user-images.githubusercontent.com/50607317/192440995-a102d38d-27fb-4b66-b8ee-3896cbc1996b.png)

@junoyoon @maoyubin

**Comment by imbyungjun** [comment]
I don't understand what you explained about the for loop.
However, if you are recording multiple tests, make sure all the record target methods are annotated with `@Test` like...
```groovy
@Test
public void request_fengxiang() {
    // Do something
}

@Test
public void request_fengguan() {
    // Do something
}

@Test
public void request_saoma() {
    // Do something
}
...
```
